src/ctia/stores/es/crud.clj

```diff
@@ -7,7 +7,7 @@
     :refer [allow-read? allow-write? restricted-read?]]
    [ctia.lib.pagination :refer [list-response-schema]]
    [ctia.schemas.core :refer [ConcreteSortExtension]]
-   [ctia.schemas.search-agg
+   [ctia.schemas.search-agg :as ss
```

Comment:
What do you think about more explicit `search-schemas`?  Or if it too long maybe just add one more entry to `:refer`, anyway there are already enough )